test/unit/operation_test.js

```diff
@@ -19,9 +19,9 @@ describe('Operation', function() {
 
     describe(".payment()", function () {
         it("creates a paymentOp", function () {
-            var destination = "gspbxqXqEUZkiCCEFFCN9Vu4FLucdjLLdLcsV6E82Qc1T7ehsTC";
+            var destination = "GB7ANT5Q6PZ6KJGAXGQ63ZX3TM7BBLANJTUJ4SQOAHRVWQL6HNVJPYLZ";
```

Comment:
You've changed the destination in this test to a different value than the `createAccount()` test that used the same address before.
![switch_keys_from_base58check_to_strcheck__base32___crc16-xmodem__by_bartekn_ _pull_request__15_ _stellar_js-stellar-base](https://cloud.githubusercontent.com/assets/366/8829858/d5553630-304d-11e5-9858-158cecd0b20f.png)

Is that intentional?